src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -277,10 +278,10 @@ public float GetPredationScore(Species species, Species preySpecies, IBiomeCondi
             engulfScore = catchScore * Constants.AUTO_EVO_ENGULF_PREDATION_SCORE;
         }
 
-        var (pilusScore, oxytoxyScore, mucilageScore) = GetPredationToolsRawScores(microbeSpecies);
-
-        // TODO: Support mucilage in predation score
-        mucilageScore *= 0;
+        var (pilusScore, oxytoxyScore, predatorSlimeJetScore) = GetPredationToolsRawScores(microbeSpecies);
+        var (_, _, preySlimeJetScore) = GetPredationToolsRawScores(prey);
+        var slimeJetScore = predatorSlimeJetScore - preySlimeJetScore;
+        slimeJetScore = Math.Max(slimeJetScore, 0);
```

Comment:
I'm not actually sure there is any reason score can't be negative. It would probably be good to penalize predation versus prey with slime jets